src/main/resources/templates/vue_templates/components/sidebar.vue

```diff
@@ -4,13 +4,13 @@
             <div class="sidebar-part">
                 <div class="sidebar-header">
                     <div class="logo_section">
-                        <a href="/"><img class="logo_icon img-responsive" src="/images/gaia.png" alt="#"></a>
+                        <a href="/"><img class="logo_icon img-responsive" src="/static/images/gaia.png" alt="#"></a>
```

Comment:
Percy says tha tthe main logo doesn't shows up anymore.
I have the same issue.

![image](https://user-images.githubusercontent.com/7531844/73122104-33c37500-3f81-11ea-9eb0-3a0f955233f7.png)

Reply:
thanks IDE when refactoring.... :joy: